frontend/src/components/donation_page/index.css

```diff
@@ -0,0 +1,16 @@
+.content-styling {
+  font-size: 32px;
```

Comment:
this looks a little big to me? Maybe we go with 24px?

Reply:
Done